hw/ip/flash_ctrl/flash_ctrl.core

```diff
@@ -17,6 +17,7 @@ filesets:
       - lowrisc:prim:lc_sync
       - lowrisc:ip:flash_ctrl_pkg
       - "fileset_top ? (lowrisc:systems:flash_ctrl)"
+      - "fileset_topgen ? (lowrisc:systems:topgen)"
```

Comment:
are the quotes necessary here?

Reply:
yeah this was recommended by @imphil that for all conditionals it's best to keep them in quotes. I may have even filed an issue somewhere...

Reply:
ah ok, thanks for clarifying.